coral/pkg/graphs/resource_models/Maritime Vessel.json

```diff
@@ -3437,21 +3437,6 @@
                     "visible": false,
                     "widget_id": "10000000-0000-0000-0000-000000000002"
                 },
-                {
-                    "card_id": "9f07bc4a-f457-11eb-bbb8-a87eeabdefba",
-                    "config": {
-                        "label": "National Grid Reference",
-                        "placeholder": "Enter the centre point map reference of the resource."
-                    },
-                    "id": "6b45ff6d-2dc3-11ec-a2ea-a87eeabdefba",
-                    "label": {
-                        "en": "National Grid Reference"
-                    },
-                    "node_id": "9f07e457-f457-11eb-b218-a87eeabdefba",
-                    "sortorder": null,
-                    "visible": true,
-                    "widget_id": "bcae8e90-09f7-4ae3-906b-7c7bb71a6ddf"
-                },
```

Comment:
I don't know why the tm65 card hasn't been explicitly added for Maritime Vessel json. It is able to work correctly just using the default node stated in tm65centrepoint.py 
[maritime-example.webm](https://github.com/flaxandteal/coral-arches/assets/104827559/338b959d-17a1-4b3c-943a-b0f6615465cb)